examples/vm_snapshot_special_cleanup.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Example - delete all snapshots with label "TEST" and type "USER".
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  environment:
+    SC_HOST: "your_host"
+    SC_USERNAME: "your_username"
+    SC_PASSWORD: "your_password"
+
+  tasks:
+  # ------------------------------------------------------
+    - name: List all snapshots
+      scale_computing.hypercore.vm_snapshot_info:
+      register: snapshot_results
+
+    - name: Create filtered_snapshots list
+      set_fact:
+        filtered_snapshots: []
+
+    - name: Loop through snapshots and add snapshots with label 'TEST' and type 'USER' to filtered_snapshots
+      set_fact:
+        filtered_snapshots: "{{ filtered_snapshots + [ item ] }}"
+      when: "{{ item.label  == 'TEST' and item.type == 'USER' }}"
```

Comment:
Make `'TEST'` a variable with name like `snapshot_label`. Then example can actually be used.